pkgs/development/libraries/ncurses/default.nix

```diff
@@ -48,20 +48,17 @@ stdenv.mkDerivation (finalAttrs: {
         "/usr/share/terminfo" # upstream default, probably all FHS-based distros
         "/run/current-system/sw/share/terminfo" # NixOS
       ]}"
-  ];
+  ] ++ lib.optional (stdenv.buildPlatform != stdenv.hostPlatform) "--with-build-cc=${stdenv.cc.targetPrefix}cc";
```

Comment:
Wouldn't this be the host C compiler?  The build compiler would be just cc.  (Or is this a terminology mismatch between Nixpkgs and ncurses?)

Reply:
It's definitely confusing. See https://github.com/mirror/ncurses/blob/master/INSTALL#L2421

Reply:
Could maybe use a comment then.

Reply:
> actually had it set to cc and it failed during configure

because there's no cc unless the for build cc is in depsBuildBuild

Reply:
and indeed `pkgsCross.aarch64-multiplatform.ncurses` fails

```
ncurses-aarch64-unknown-linux-gnu> checking if the build-compiler "aarch64-unknown-linux-gnu-cc" works... no
```

Reply:
I pulled that out of depsBuildBuild to get rid of the mismatch warning so I'm not sure what the fix would be.

Reply:
Oh! My nixos install had binfmt enabled and that's why it succeeded for me.